### DISCUSSION TITLE
Dvorak + Voyager

### DISCUSSION THREAD

**Discussion by h2ogeek** [manna-harbour/miryoku#307] [Q&A] [answered]
Any recommendation about using Miryoku with Dvorak + Voyager?

**Comment by manna-harbour** [comment, marked as answer]
For voyager see https://github.com/manna-harbour/miryoku/discussions/241.

For Dvorak on Miryoku QMK see https://github.com/manna-harbour/miryoku/tree/master/docs/reference#alternative-layouts. For Dvorak on Miryoku Oryx see https://github.com/manna-harbour/miryoku/tree/master/data/oryx#implementation-notes.
